ControlPanel.java

```diff
@@ -0,0 +1,61 @@
+public class ControlPanel{
+    Setter scanner = new Setter();
```

Comment:
still have multiple scanner what are you hiding in this class, not acceptable solution, use function parameters instead!